PURL-TYPES.rst

```diff
@@ -413,6 +441,7 @@ Other candidate types to define:
 - ``nim`` for Nim packages:
 - ``nix`` for Nixos packages:
 - ``opam`` for OCaml packages:
+- ``os`` for operating system distributions:
```

Comment:
Sounds like this proposal is not for operating system distributions, but flat-out operating system descriptors.